playbooks/tasks/portal-configs-get.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+# Get portal configs
+# - Load portal configs from secrets backend (e.g. LastPass)
+# - Set default values (for unset keys)
+# - Set mongodb flag
+# - Set mongodb config (if mongodb is on)
+# - Set accounts flag
+
+- name: Include loading portal config
+  include_tasks: "{{ playbook_dir }}/{{ load_portal_config_handler }}"
+
+- name: Set portal config to empty (if it hasn't yet been saved to secrets backend)
+  set_fact:
+    webportal_server_config: "{{ webportal_server_config | default({}) }}"
+
+- name: Set portal config missing variables to default values
+  set_fact:
+    webportal_server_config: "{{ webportal_server_config | combine({item.key: item.value}) }}"
+  when: webportal_server_config[item.key] is not defined
+  loop: "{{ webportal_server_config_defaults | dict2items }}"
+  no_log: True
+
+- name: Set MongoDB flag
+  set_fact:
+    portal_mongodb_on: "{{ webportal_server_config.portal_modules is defined and webportal_server_config.portal_modules is not none and ('m' in webportal_server_config.portal_modules or 'a' in webportal_server_config.portal_modules) }}"
+
+# Get MongoDB config (if MongoDB is on)
+- block:
+    - name: Set MongoDB config keys
+      set_fact:
+        mongodb_config_keys:
+          - skynet_db_user
+          - skynet_db_pass
+
+    - name: Set MongoDB config (server config can override common config)
```

Comment:
same, parathesis in the print out aren't as helpful as a nice detailed comment imo.